File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q05.py
def main():
    #Input
    num = int(input("Enter a number with 3 digits: "))

    #Processing
    hundreds = round(num / 100)
    dozens = round((num % 100) / 10)
    units = round((num % 100) % 10)

    sum = hundreds + dozens + units

    #Output 
    print(f"The sum of H + D + U is equal to: {sum}")


main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/While/lista_fabio/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q05.js
import { question } from "readline-sync";
function main(){
    //Entrada
    const num = parseInt(question("Enter a number with 3 digits: "))

    //Processamento
    const hundreds = Math.floor(num / 100)
    const dozens = Math.floor((num % 100) / 10)
    const units = (num % 100) % 10
    const sum = hundreds + dozens + units 

    //Saída
    console.log(`The sum of H + D + U is equal to: ${sum}`)

}

main()